kube/acceptbitcoincash-deployment.yml

```diff
@@ -0,0 +1,23 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  namespace: default
+  labels:
+    service: acceptbitcoincash
+  name: acceptbitcoincash
+spec:
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        service: acceptbitcoincash
+    spec:
+      containers:
+      - image: zquestz/acceptbitcoincash
```

Comment:
is this the case even on our repo?

Reply:
For now, I plan on updating it when you have your own docker hub account. Right now the pre-built one is on my account. This is temporary.